.github/workflows/label-notify.yml

```
@@ -0,0 +1,16 @@
+name: "Notify users based on issue labels"
+
+on:
+  issues:
+      types: [labeled]
+
+jobs:
+  notify:
+    runs-on: ubuntu-latest
+    steps:
+        - uses: jenschelkopf/issue-label-notification-action@1.3
```

For third party actions I think we should pin the precise SHA since tags can be updated under us.

```suggestion
        - uses: jenschelkopf/issue-label-notification-action@f7d2363e5efa18b8aeea671ca8093e183ae8f218 # 1.3
```

---

Of course. Thanks for highlighting!